php/libraries/LorisForm.class.inc

```diff
@@ -119,7 +120,7 @@ class LorisForm
      *                        get added. Currently only the "class"
      *                        attribute gets added.
      *
-     * @return &array A reference to the array that was added to $this->form
+     * @return array A reference to the array that was added to $this->form
```

Comment:
These changes are all wrong. The function(s) are returning references to arrays, not array values.